CHANGELOG.md

```diff
@@ -38,6 +38,11 @@
 
 ## [2.0.0] - TBA
 
+#### Breaking changes
```

Comment:
This doesn't seem like a breaking change, should this be under `#### Under The Hood`?

Reply:
Assuming interface change is breaking changes.

Reply:
talked about this in zoom, going to reword to "under the hood"